src/Catrobat/StatusCode.php

```diff
@@ -41,6 +41,8 @@ class StatusCode
   const PASSWORD_INVALID = 532;
   const CREDITS_TO_LONG = 833;
 
+  const ALREADY_STOLE = 510;
```

Comment:
try not to override codes that are already defined https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/510